server.go

```diff
@@ -125,17 +96,22 @@ type Server struct {
 	// Res field of the Session you can write an error response
 	// to the client.
 	//
-	// The boolean returned indicates whether the returned subscription
-	// is valid or not. If it is valid, the Provider will receive it
-	// and events will be sent to this client, otherwise the request
-	// will be ended.
+	// The boolean returned indicates whether the given request
+	// should be accepted or not. If it is true, the Provider will receive
+	// a new subscription for the connection and events will be sent
+	// to this client, otherwise the request will be ended.
+	//
+	//
```

Comment:
Dumb nitpick but here we are
```suggestion
	//
```